Decisions.TruBot/Data/TruBotAssignmentHelper.cs

```diff
@@ -0,0 +1,75 @@
+using DecisionsFramework;
+using DecisionsFramework.Data.ORMapper;
+using DecisionsFramework.ServiceLayer.Services.Assignments;
+using DecisionsFramework.ServiceLayer.Services.Folder;
+using DecisionsFramework.ServiceLayer.Utilities;
+
+namespace Decisions.TruBot.Data
+{
+    public class TruBotAssignmentHelper
+    {
+        private static Log log = new Log("TruBot Assignments");
+
+        static ORM<TruBotProcess> botProcessOrm = new();
+        
+        public static void CreateAssignment(TruBotProcess truBotProcess, string interfaceId = TruBotConstants.TRUBOT_ASSIGNMENT_FOLDER_ID)
```

Comment:
@jeff-decisions and @shawn-lirette - I'm not sure why we have to use Assignments at all, they seem heavy for this. What was the reason for choosing them?

Reply:
The intention was to give the user a way to check and update the status of a process manually

Reply:
We only need to track bot progress, this should be an Entity I think.
No individual is responsible for monitoring these as far as I know, the Assignment notes only came from their suggestion to use the WOES step.

Reply:
Okay, so I should essentially remove this since the process is already being tracked automatically elsewhere then?